src/components/ShareTrajectoryModal/EmbedSnippetPanel.tsx

```diff
@@ -104,18 +104,18 @@ const EmbedSnippetPanel = ({ startTime }: EmbedSnippetPanelProps) => {
                             <VerticalFlexbox gap={24}>
                                 <HorizontalFlexbox gap={8} alignItems="center">
                                     <Input
-                                        defaultValue={height}
+                                        value={width}
                                         className={styles.numberInputs}
                                         onChange={(e) => {
-                                            handleChangeHeight(e.target.value);
+                                            handleChangeWidth(e.target.value);
                                         }}
                                     />
                                     <span> x </span>
                                     <Input
-                                        value={width}
+                                        defaultValue={height}
```

Comment:
switching these made me notice that one of these has value and one as defaultValue. seems to be working, but should be consistent